sap_hana/tests/test_integration.py

```diff
@@ -61,6 +57,21 @@ def _run_until_stable(dd_run_check, check, aggregator):
         retries -= 1
 
 
+def _assert_standard_metrics(aggregator, instance):
+    # Not all metrics are present in every check run
+    missing_metrics = []
+    for metric in metrics.STANDARD:
+        if metric in aggregator.metric_names:
+            aggregator.assert_metric_has_tag(metric, 'server:{}'.format(instance['server']))
+            aggregator.assert_metric_has_tag(metric, 'port:{}'.format(instance['port']))
+        else:
+            missing_metrics.append(metric)
+    assert len(missing_metrics) / len(metrics.STANDARD) < 0.1, 'Missing metrics: %s\nPresent metrics: %s' % (
```

Comment:
Quite a low percentage

Reply:
Sorry. I was understanding it as a percentage, not realising that it is not treated as such. Converted to percentage, we are talking about 10%, which seems fine to me.